metrics.md

```diff
@@ -572,6 +572,57 @@ Choice elements:
 Notes:
 - These are the choice fields in US Core, but could use the FHIR definitions to create a full list, or could do a more general query based on JSON key prefix (not supported by all databases and potentially slow even where supported)
 
+### c_attachment_count
+**[structure]** Survey of attachment data and formats
```

Comment:
We don't use the term "survey" anywhere else, though I'm not necessarily averse to it. For consistency, maybe "Count of attachments by format" instead?